tensorflow/compiler/xla/service/gpu/BUILD

```diff
@@ -1223,6 +1223,37 @@ tf_cc_test(
     ],
 )
 
+cc_library(
+    name = "fusion_bitcast_lift",
+    srcs = ["fusion_bitcast_lift.cc"],
+    hdrs = ["fusion_bitcast_lift.h"],
+    deps = [
+        "//tensorflow/compiler/xla:shape_util",
+        "//tensorflow/compiler/xla/service:hlo",
+        "//tensorflow/compiler/xla/service:hlo_casting_utils",
```

Comment:
This target was removed yesterday, and depending on "//tensorflow/compiler/xla/service:hlo" is enough to get access to hlo_casting_utils.h header.
Can you please remove this line? This should also show up as compile error if you sync to HEAD.

Reply:
I rebased and removed that line. Now it build and the test still pass.